### ISSUE TITLE
continuation indent size

### ISSUE THREAD

**Issue by cybermerlin** [eslint/eslint#10871]
hi
sorry for quickly offer...

Just look in https://github.com/editorconfig/editorconfig-jetbrains/issues/57#issuecomment-275606421

**Comment by cybermerlin** [comment]
eslint latest.
it's the new rule.
just look in the link.
I'm not sure about more info, because it's just a `continuation_ident_size`. Sorry.

**Comment by ilyavolodin** [comment]
@cybermerlin Please fill out our bug or proposal template. It's very unclear what you are reporting. The link doesn't really contain any relevant information that might shine more light on it.

**Comment by cybermerlin** [comment]
```js
function run(){
  let multiple = {
      continuation_ident_size_property: 'asdfsf asdfdsf\
          asdfdf'
  };
}
```
line 1: zero symbols befofre
line 2: +2 space = like ident size
line 3: +4 space = because continuation ident size
line 4: +4 space = new continuation ident size line
@ilyavolodin, that is enough to understand?
